public/app/components/viewByFile/viewByFileTemplate.html

```diff
@@ -3,4 +3,6 @@
 	<div class="img-view-container" ng-switch-when="image">
 		<img kb-full-image-view class="img-view" ng-src="{{ viewByFileCtrl.imageUri }}">
 	</div>
+	<kb-font-viewer class="font-view-container" ng-switch-when="font" kb-open-font="viewByFileCtrl.fileInfo">
```

Comment:
I guess this is just something you copy pasted from the element above this one, but I'll say it anyway :-)

I think the class `font-view-container` is not needed here. Since you created an element directive for this, you can use this in your `scss`:

```scss
kb-font-viewer {
  ...
}
```
So the class isn't giving you extra functionality.